packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -212,6 +212,16 @@ export const hideVisually = (): SerializedStyles => css`
   width: 1px;
 `;
 
+/**
+ * Easily center content in a block
```

Comment:
We try to avoid opinionated words such as "easily" or "straight-forward" in our documentation since we don't know the context of the person reading them.

The description you wrote for the Storybook docs is nice and objective: 
```suggestion
 * Centers the content horizontally and vertically.
```